seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -5586,138 +5586,126 @@
+        "The function call looks like:",
+        "<code>parseInt(string, radix);</code>",
+        "And here's an example:",
+        "<code>var a = parseInt(\"11\", 2);</code>",
+        "The radix variable says that \"11\" is in the binary system, or base 2. This example converts the string \"11\" to an integer 3.",
+        "<hr>",
+        "Use <code>parseInt()</code> in the <code>convertToInteger</code> function so it converts a binary number to an integer and returns it."
       ],
       "challengeSeed": [
         "function convertToInteger(str) {",
-        "",
+        "  ",
         "}",
-        "convertToInteger(\"56\");"
+        "convertToInteger(\"10011\");"
       ],
       "tail": [],
       "solutions": [],
       "tests": [
-        "assert(typeof(convertToInteger(\"56\")) === \"number\", 'message: <code>convertToInteger(\"56\")</code> should return a number');",
-        "assert(convertToInteger(\"56\") === 56, 'message: <code>convertToInteger(\"56\")</code> should return 56');",
-        "assert(convertToInteger(\"77\") === 77, 'message: <code>convertToInteger(\"56\")</code> should return 77');",
-        "assert(!isNaN(convertToInteger(\"JamesBond\")); , 'message: <code>convertToInteger(\"JamesBond\")</code> should return NaN');"
+        "assert(/parseInt/g.test(code), 'message: <code>convertToInteger</code> should use the <code>parseInt()</code> function');",
+        "assert(typeof(convertToInteger(\"10011\")) === \"number\", 'message: <code>convertToInteger(\"10011\")</code> should return a number');",
+        "assert(convertToInteger(\"10011\") === 19, 'message: <code>convertToInteger(\"10011\")</code> should return 19');",
+        "assert(convertToInteger(\"111001\") === 57, 'message: <code>convertToInteger(\"111001\")</code> should return 57');",
+        "assert.isNaN(convertToInteger(\"JamesBond\"), 'message: <code>convertToInteger(\"JamesBond\")</code> should return NaN');"
       ],
       "type": "waypoint",
       "challengeType": 1,
       "translations": {}
     },
     {
-      "id": "587d7b7e367417b2b2512b22",
-      "title": "Use the parseInt function with a radix",
+      "id": "587d7b7e367417b2b2512b24",
+      "title": "Use the Conditional (Ternary) Operator",
       "description": [
-        "The parseInt() function parses a string and returns an integer. The radix variable defines the number system to convert from.",
-        "The function call looks like:",
-        "parseInt(string, radix);",
-        "var a = parseInt(\"11\", 2);",
-        "Here the radix variable defines that \"11\" is in binary system.",
-        "The above function will convert the string \"11\" to an integer 3.",
-        "Instructions",
-        "Using the parseInt() function, write a function that converts binary numbers to integers and returns them."
+        "The <dfn>conditional operator</dfn>, also called the <dfn>ternary operator</dfn>, is a one line if-else statement.",
```

Comment:
Actually, this is an expression. This is what allows it to be used in so many more places than a regular if-else statement.

Either we add "(...), is **like** a one line if-else statement", or even clearer, "(...), **can be used as** a one line if-else expression".